Description: Drop curl/types.h include as it doesn't exist anymore.
Author: Felix Geyer <debfx-pkg@fobos.de>

--- vmware-view-open-client-4.5.0-297975+dfsg.orig/lib/bora/basicHttp/http.c
+++ vmware-view-open-client-4.5.0-297975+dfsg/lib/bora/basicHttp/http.c
@@ -40,7 +40,6 @@
 #include "requestQueue.h"
 
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h>
 #include <curl/multi.h>
 
